plugins/extensions/plugins/RocketsPlugin.cpp

```diff
@@ -89,11 +89,13 @@ std::string hyphenatedToCamelCase(const std::string& hyphenated)
 
 namespace brayns
 {
+RocketsPlugin* RocketsPlugin::_instance{nullptr};
```

Comment:
global variables are evil! ;-) Please find a cleaner way to pass the RocketsPlugin to whoever needs it.